2020-10-13-all.html

```diff
@@ -53,15 +53,7 @@ <h1>All Techniques</h1>
                 <aside class="box">
                     <header class="box-h ">Summary</header>
                     <div class="box-i">
-                        <p><em>Techniques</em> are examples of ways to make web content more accessible. Most Techniques relate to a specific WCAG criterion, some offer general advice (see <a href="alt-index.html">“The Documents”</a> for an overview of different
-                            types of Techniques).
-
-                            <div class="doc-note-box">
-                                <p><strong>Editor's note</strong> some have asked for this summary to be more extensive, suggestions for what must be contained in this introduction are welcomed</p>
-
-                            </div>
-
-                        </p>
+                        <p><em>Techniques</em> describe ways to make your content more accessible. Most Techniques relate to meeting specific WCAG success criteria, some offer general advice. <a href="2020-10-20.html">About Techniques</a> provides further background, and <a href="alt-index.html">All WCAG 2 Guidance</a> provides an overview on related guidance.</p>
```

Comment:
I like: “success criteria”, as there may be multiple, and the links to About and All Guidance for context.

I did really like  to have the word “examples”, but I think without it also works and makes it shorter.

Great!